bundles/com.salesforce.b2eclipse.jdt.ls/src/main/java/com/salesforce/b2eclipse/managers/B2EPreferncesManager.java

```diff
@@ -76,7 +76,7 @@ private B2EPreferncesManager() {
     }
 
     public void setConfiguration(Map<String, Object> configuration) {
-        importBazelEnabled = MapFlattener.getBoolean(configuration, IMPORT_BAZEL_ENABLED, false);
+        importBazelEnabled = MapFlattener.getBoolean(configuration, IMPORT_BAZEL_ENABLED, true);
         importBazelSrcPath = MapFlattener.getString(configuration, BAZEL_SRC_PATH, BAZEL_DEFAULT_SRC_PATH);
         importBazelTestPath = MapFlattener.getString(configuration, BAZEL_TEST_PATH, BAZEL_DEFAULT_TEST_PATH);
```

Comment:
@plaird If the new support in 1.5 to be more flexible about source paths can this preference/setting be removed?

Reply:
Good Q. Looking at the LS code, I am not sure why this was an option. It seems to block importing a Bazel workspace into LS?

Reply:
Ah, ok. So, the short answer is no, it isn't necessary for the user to tell us where the source is. We use Bazel query to figure that out if they aren't using Maven structure.

The longer answer is we may still want some mechanism for a user to provide this, for performance reasons. Bazel query is expensive. If the user just knows that many of their packages are laid out like source/dev/java/... then we should accept their help. But it wouldn't be a simple pref like this. I am thinking the user would provide a set of package 'templates' that we could try, before we resort to bazel query. We have one built-in template: Maven structure. Each template would have main/test source locations.

Reply:
@guw , @plaird it will be removed within the import process consolidation